### PR TITLE
Spellcheck fixes in release notes templates

### PR DESCRIPTION
**Pull request by peterjunpark**
https://github.com/ROCm/rocm-docs-core/pull/897

### CODE REVIEW

File: .wordlist.txt [hunk 1]
```diff
@@ -26,10 +26,10 @@ ATI
 AddressSanitizer
 AlexNet
 Arb
+Autocast
```
Comment (peterjunpark) [inline .wordlist.txt:29]: https://pytorch.org/docs/stable/amp.html#autocasting

File: .wordlist.txt [hunk 2]
```diff
@@ -67,6 +67,7 @@ CommonMark
 Concretized
 Conda
 ConnectX
+CuPy
```
Comment (peterjunpark) [inline .wordlist.txt:70]: https://cupy.dev/

File: .wordlist.txt [hunk 3]
```diff
@@ -86,13 +87,15 @@ DataLoader
...
 Dependabot
+Deprecations
 DevCap
 Dockerfile
 Doxygen
 ELMo
 ENDPGM
 EPYC
 ESXi
+EoS
```
Comment (peterjunpark) [inline .wordlist.txt:98]: end of support

File: .wordlist.txt [hunk 4]
```diff
@@ -143,6 +146,7 @@ HPCG
 HPE
 HPL
 HSA
+HW
```
Comment (peterjunpark) [inline .wordlist.txt:149]: abbreviation for 'hardware'

File: .wordlist.txt [hunk 5]
```diff
@@ -166,6 +170,7 @@ ImageNet
 InfiniBand
 Inlines
 IntelliSense
+Interop
```
Comment (peterjunpark) [inline .wordlist.txt:173]: 'interoperation' or 'interoperability'

File: .wordlist.txt [hunk 6]
```diff
@@ -208,6 +213,7 @@ MVFFR
 Makefile
 Makefiles
 Matplotlib
+Matrox
```
Comment (peterjunpark) [inline .wordlist.txt:216]: specific third-party graphics card

File: .wordlist.txt [hunk 7]
```diff
@@ -274,6 +280,7 @@ PerfDb
 Perfetto
 PipelineParallel
 PnP
+PowerEdge
```
Comment (peterjunpark) [inline .wordlist.txt:283]: specific third-party server

File: .wordlist.txt [hunk 8]
```diff
@@ -478,6 +489,8 @@ denoise
 denoised
 denoises
 denormalize
+dequantization
+dequantizes
```
Comment (peterjunpark) [inline .wordlist.txt:493]: https://docs.nvidia.com/deeplearning/tensorrt/operators/docs/Dequantize.html

File: .wordlist.txt [hunk 9]
```diff
@@ -455,8 +464,10 @@ composable
 concretization
 config
 conformant
+constructible
 convolutional
 convolves
+copyable
```
Comment (peterjunpark) [inline .wordlist.txt:470]: ability to be copied

File: .wordlist.txt [hunk 10]
```diff
@@ -455,8 +464,10 @@ composable
 concretization
 config
 conformant
+constructible
```
Comment (peterjunpark) [inline .wordlist.txt:467]: ability to be constructed

File: .wordlist.txt [hunk 11]
```diff
@@ -489,8 +502,9 @@ distro
 el
 embeddings
 enablement
-endpgm
 encodings
+endpgm
+enqueue
```
Comment (peterjunpark) [inline .wordlist.txt:507]: to push onto a queue

File: .wordlist.txt [hunk 12]
```diff
@@ -501,13 +515,15 @@ ffmpeg
 filesystem
 fortran
 fp
+gRPC
```
Comment (peterjunpark) [inline .wordlist.txt:518]: https://en.wikipedia.org/wiki/GRPC

File: .wordlist.txt [hunk 13]
```diff
@@ -501,13 +515,15 @@ ffmpeg
...
 galb
 gcc
 gdb
 gfortran
 gfx
 githooks
 github
+globals
```
Comment (peterjunpark) [inline .wordlist.txt:526]: probably referring to global variables

File: .wordlist.txt [hunk 14]
```diff
@@ -574,6 +591,7 @@ mivisionx
 mkdir
 mlirmiopen
 mtypes
+mutex
```
Comment (peterjunpark) [inline .wordlist.txt:594]: common abbreviation for mutual exclusion

File: .wordlist.txt [hunk 15]
```diff
@@ -596,26 +614,35 @@ pragma
 pre
 prebuilt
 precompiled
+preconditioner
 preconfigured
 prefetch
 prefetchable
 prefill
 prefills
+preloaded
```
Comment (peterjunpark) [inline .wordlist.txt:623]: loaded beforehand

File: .wordlist.txt [hunk 16]
```diff
@@ -668,11 +697,14 @@ spack
...
 stochastically
 strided
+subcommand
 subdirectory
 subexpression
 subfolder
 subfolders
+submodule
 supercomputing
+symlink
```
Comment (peterjunpark) [inline .wordlist.txt:707]: most common way to saying symbolic link a.k.a. soft link

File: .wordlist.txt [hunk 17]
```diff
@@ -713,10 +746,13 @@ vectorized
 vectorizer
 vectorizes
 vjxb
+voxel
```
Comment (peterjunpark) [inline .wordlist.txt:749]: 3D "pixel"

File: .wordlist.txt [hunk 18]
```diff
@@ -258,6 +266,7 @@ OpenMP
 OpenMPI
 OpenSSL
 OpenVX
+OpenXLA
```
Comment (peterjunpark) [inline .wordlist.txt:269]: https://github.com/openxla

File: .wordlist.txt [hunk 19]
```diff
@@ -284,7 +294,10 @@ RCCL
 RDC
 RDMA
 RDNA
+README
 RHEL
+RNN
```
Comment (peterjunpark) [inline .wordlist.txt:299]: recurrent neural network

File: .wordlist.txt [hunk 20]
```diff
@@ -296,6 +309,7 @@ ROCm
 ROCmCC
 ROCmSoftwarePlatform
 ROCmValidationSuite
+ROCprofiler
```
Comment (peterjunpark) [inline .wordlist.txt:312]: as in ROCprofiler-SDK

File: .wordlist.txt [hunk 21]
```diff
@@ -537,6 +559,7 @@ hpp
 hsa
 hsakmt
 hyperparameter
+iDRAC
```
Comment (peterjunpark) [inline .wordlist.txt:562]: product name

**Review by samjwu — APPROVED** [review]

**Pull request merged**